fail2ban/protocol.py

```diff
@@ -55,6 +55,7 @@
 ["start <JAIL>", "starts the jail <JAIL>"], 
 ["stop <JAIL>", "stops the jail <JAIL>. The jail is removed"], 
 ["status <JAIL>", "gets the current status of <JAIL>"],
+["status <JAIL> extended", "gets the current status of <JAIL> with extended info"],
```

Comment:
Also to keep in mind, my ultimate goal is to have fail2ban-client status <JAIL> ??? mapped to munin graphs, which need to know how to call fail2ban-client as well as which line to parse (ASN, Country, etc). One use-case may be with plugin names: fail2ban_basic (current status, counts per jail), fail2ban_cyrmu_asn, fail2ban_cyrmu_rir, fail2ban_XXX_YYY - where munin calls fail2ban-client status <JAIL> XXX and parses lines dealing with YYY (code determines)